Lib/test/test_io.py

```diff
@@ -807,8 +807,8 @@ def test_multi_close(self):
         self.assertRaises(ValueError, f.flush)
 
     def test_RawIOBase_read(self):
-        # Exercise the default RawIOBase.read() implementation (which calls
-        # readinto() internally).
+        # Exercise the default limited RawIOBase.read() implementation (which
```

Comment:
I dislike "limited", I would prefer to write read(n) for "limited", and read() for "unlimited".